wagtail/admin/wagtail_hooks.py

```diff
@@ -974,6 +976,7 @@ def register_icons(icons):
         "help.svg",
         "history.svg",
         "home.svg",
+        "horizontal-dots.svg",
```

Comment:
nitpick: let's be consistent with the naming - `dots-vertical.svg` and `dots-horizontal.svg` similar to the `doc-*.svg` icons